backend/app/modules/imagefetcher/FetcherActor.scala

```diff
@@ -23,36 +19,49 @@ object FetcherActor {
   /**
     * Messages definitions
     */
-  case class FetchMessage(start: String, end: String, position: JsValue)
+  case class FetchRGB(start: String, delta: String, scale: Double = 1.0, polygon: JsValue)
```

Comment:
Is there a way to NOT provide a scale if it is left undefined by the caller? I mean having a default value on the actor and the server is kinda useless and potentially dangerous ("hey, I modified the default, why isn't this working?!")?